Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2654,6 +2701,7 @@ namespace OpenInfraPlatform
 						throw oip::InconsistentGeometryException(curve, "TrimmingSelect is wrong!");

```

Comment:
Just noticed (but not your mistake) - this function takes `IfcCurveMeasureSelect` - but the error speaks about trimmings. This should be normalized.

Reply:
Resolved: 4192532